  <launch>

  <!-- Detect Ball -->
  <node name="detect_ball" pkg="coconuts_master" type="detect_ball">
    <param name="H_MIN" type="int" value="0" />
    <param name="H_MAX" type="int" value="50" />
    <param name="S_MIN" type="int" value="120" />
</node>


<!-- Publish and Sync -->
<node name= "publishInfo" pkg="coconuts_master" type="publishInfo"/>

<!-- Disparity Map Analysis-->
<node name= "getDistance" pkg="coconuts_master" type="getDistance"/>

<!--stereoimageProc-->
<node ns="stereo" name= "stereo_image_proc" pkg="stereo_image_proc" type="stereo_image_proc">
    <param name="min_disparity" type="int" value="-42" />

</node>

<!-- usb_cam -->
<node name= "usb_cam" pkg="usb_cam" type="usb_cam_node" >
<param name="video_device" type="string" value="/dev/video0"/>
</node>

  <node ns="camera_2" pkg="uvc_camera" type="uvc_camera_node" name="uvc_camera_node_2" output="screen">
    <param name="width" type="int" value="640" />
    <param name="height" type="int" value="480" />
    <param name="fps" type="int" value="30" />
    <param name="frame" type="string" value="webcam" />
    <param name="device" type="string" value="/dev/video1" />
    <param name="auto_focus" type="bool" value="0" />
  </node>



  </launch>
